### ISSUE TITLE
[Legitimate Site Unblocked] platinumcryptoacademy.com

### ISSUE THREAD

**Issue by nick26pca** [MetaMask/eth-phishing-detect#12223]
**Domain:https://www.platinumcryptoacademy.com/**

**Details:Crypto Trading Academy website and Free Crypto Trading Tips Magazine - Cryptonaire Weekly** *Some details about the domain (optional)*

- [x] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

There was a phishing attack on our website last week, we have already with immediate effect removed all suspicious and infected files, please unblock our site as we have reviewed and no further infected files are present on our website.

Thank you

**Comment by deshvin** [comment]
Site not present in blocklist; doesn't appear to be blocked from my end

**Comment by nick26pca** [comment]
Hi @deshvin Thank you for reviewing, but whenever any user who has Metamask installed, which is almost all our users since it is a Crypto Education Site, all of them receive the following error and it is still present, do check as it also mentions the eth-phishing github url for getting unblocked as well, thank you.
![image](https://user-images.githubusercontent.com/129944753/234847943-ac2dc6cd-b6d7-4c7d-89f4-6d09999559f7.png)

**Comment by AlexHerman1** [comment]
@nick26pca  this is not on our blocklist, might just be a local caching issue for you. please open a new issue if you continue to run into problems.

**Comment by deshvin** [comment]
This domain was being blocked on a third-party list that we consume. I've contacted the team that handles this list, and they have removed the domain. The phishing warning should fall off shortly.
